study.md

```diff
@@ -57,7 +57,8 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another method required for persistence?
 
 ```md
-<!-- your response here -->
+CREATE
+.create both creates and saves a new record into the database
```

Comment:
`#create` will instantiate a new object and save it to the database.

`#new` instantiates a new object but doesn't save it to the database.  `#save` needs to be invoked on it afterward to save it.